scripts/build.ps1

```diff
@@ -292,6 +292,8 @@ function CMake-Build {
 #                     Main Execution                         #
 ##############################################################
 
+clog --installDirectory (Join-Path $RootDir build clog)
```

Comment:
We want to support cmake-only (no pwsh) builders. This won't help them. Should we make this part of the prepare-machine steps (and update the cmake build instructions)?

Reply:
Or make cmake do this?

Reply:
Done. Moved into cmake.